src/components/fee-row/fee-row.tsx

```diff
@@ -41,14 +39,12 @@ export function FeeRow(props: FeeRowProps): JSX.Element {
   const [isCustom, setIsCustom] = useState(false);
 
   useEffect(() => {
-    // Check for query error on mount and fallback
-    // to the custom input if needed
-    if (feeEstimationsError) {
```

Comment:
I don't think we need to pass in `feeEstimationsError` anymore bc we aren't falling back to custom input.